src/App.tsx

```diff
@@ -10,37 +12,23 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section">
-      <h1 className="title">Selected tab is Tab 1</h1>
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
```

Comment:
Use just tab, no need to pass the ID and then bother with finders. 

 `const [selectedTab, setSelectedTab] = useState(tabs[0]);`